tests/Oryx.BuildImage.Tests/NodeJSSampleAppsTest.cs

```diff
@@ -130,6 +130,162 @@ public void Build_CopiesOutput_ToNestedOutputDirectory()
                 result.GetDebugInfo());
         }
 
+        [Fact]
+        public void BuildNodeApp_ConfigureAppInsight__WithDefaultNodeVersion_AIEnvironmentVariableSet()
+        {
+            // Arrange
+            var volume = CreateWebFrontEndVolume();
+            var appDir = volume.ContainerDir;
+            var nestedOutputDir = "/tmp/output";
+            var script = new ShellScriptBuilder()
+                .AddCommand($"oryx build {appDir} -o {nestedOutputDir} --log-file {appDir}/1.log")
```

Comment:
Based on what you mentioned in scrum meeting, I think we should not rely on the log file to verify in this test as it can make this test flaky and I do not feel its a bug actually since we cannot depend on when NLog decides to flush the logs to a file